components/Layout.tsx

```diff
@@ -5,7 +5,7 @@ import Footer from './Footer'
 const Layout: React.FC = ({ children }) => (
   <>
     <AppNav />
-    <div className="container">{children}</div>
+    <div className="container-md">{children}</div>
```

Comment:
This change cascaded all these snapshot updates. It's for mobile layout, smaller screens with less than `768px` wide get more space.